src/main/index.ts

```diff
@@ -34,14 +34,22 @@ function init() {
 
     // work around chrome 66 disabling autoplay by default
     app.commandLine.appendSwitch("autoplay-policy", "no-user-gesture-required");
-
+    app.commandLine.appendSwitch(
+        "enable-features",
+        "UseOzonePlatform,VaapiVideoDecodeLinuxGL,VaapiVideoEncoder,WebRTCPipeWireCapturer,VaapiVideoDecoder"
```

Comment:
these should be obsolete
```suggestion
        "VaapiVideoDecodeLinuxGL,VaapiVideoEncoder,VaapiVideoDecoder"
```